features/buyer/buyer_journey.feature

```diff
@@ -0,0 +1,26 @@
+@not-production @functional-test
+Feature: Buyer user journey through Digital Marketplace
+
+Scenario: Check the admin frontend /_status
+  Given I am logged in as a admin user
+  Then I am on the frontend /admin/_status page
+  Then I see 'ok' as the value of the 'status' JSON field
+  And Display the value of the 'version' JSON field as 'Release version'
+
+Scenario: Setup for tests
+  Given I have test suppliers
+  And The test suppliers have live services
+  And All services for the test suppliers are Public
+  Then I take a screenshot
```

Comment:
Screenshots are used during development and then taken out usually. Unless there a reason why this one is here?
Also I don't really understand this scenario, what does this flow do? Should it be a separate scenario? Maybe what you need is "Background" feature, like here? https://github.com/alphagov/digitalmarketplace-functional-tests/blob/master/features/supplier/supplier_applies_for_an_opportunity.feature